src/ReleaseHistory.md

```diff
@@ -1,10 +1,11 @@
 # BinSkim Release History
 
-## Unreleased
+## **v2.0.0** [NuGet Package](https://www.nuget.org/packages/Microsoft.CodeAnalysis.BinSkim/2.0.0)
```

Comment:
>## **v2.0.0** [](http://example.com/codeflow?start=0&length=13)

Note on versioning. This change includes the sarif-sdk changes related to newtonsoft and when we updated BinSkim to use Newtonsoft 13.0.1, we neglected to take the change in major version at that time. This version number more accurately reflects our standard in versioning.

Should we elect to adjust versioning, the absolute minimum version acceptable to be argued would be 1.10.0, given that this includes new rules and is not strictly bug fixes.

Documenting versioning rationale in this PR comment for anyone onboarding to the BinSkim release process.